### PR TITLE
PACT-726 Fix issue with missing record type on save and discard recor…

### PR DESCRIPTION
**Pull request by userjd**
…d pages

### CODE REVIEW

File: app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEditDiscard.scala.html [hunk 1]
```diff
@@ -35,7 +37,10 @@
 @content = {
   <h3 class="govuk-notification-banner__heading">@message</h3>
   <p class="govuk-body"><strong>@heading</strong></p>
-  <p class="govuk-body"><strong>@messages("edit-set.record.edit.id", oci)</strong></p>
+  <p class="govuk-body"><strong>@messages("edit-set.record.edit.id", oci, recordType match {
```
Comment (felixgb) [inline app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEditDiscard.scala.html:40]: Are record types mandatory? i.e. when would you have an empty record type?

File: test/views/EditSetRecordEditSaveSpec.scala [hunk 2]
```diff
@@ -62,7 +63,7 @@ class EditSetRecordEditSaveSpec extends BaseSpec {
     "render the header" in {
 
       val document = generateDocument()
-
+      println(document)
```
Comment (felixgb) [inline test/views/EditSetRecordEditSaveSpec.scala:66]: extra println here!

File: app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetRecordController.scala [hunk 3]
```diff
@@ -134,9 +135,10 @@ class EditSetRecordController @Inject() (
       val heading: String =
         resolvedMessage(MessageKeys.heading, editSetRecords.getEditSetRecordByOCI(oci).get.ccr)
       val message: String = resolvedMessage(MessageKeys.buttonDiscard)
+      val recordType: Option[RecordType] = editSetRecords.getEditSetRecordByOCI(oci).get.recordType
```
Comment (felixgb) [inline app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetRecordController.scala:138]: There's a `get` here - how do we handle the case where `getEditSetRecordByOCI` is None?

File: test/views/EditSetRecordEditSaveSpec.scala (hunk 2 above)
Comment (userjd) [reply test/views/EditSetRecordEditSaveSpec.scala]: removed

File: app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetRecordController.scala (hunk 3 above)
Comment (userjd) [reply app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetRecordController.scala:138]: Tests added to reflect the behaviour without record type suffix.

**Review by rwalpole — APPROVED** [review]

**Pull request merged**